tests/core/plugins/test_session.py

```diff
@@ -61,7 +62,13 @@ def fake_session_list(self, project_name, config):
     return ["0xdeadbeef"]
 
 
-@pytest.mark.parametrize("provider", ["docker"])
+@pytest.mark.parametrize(
```

Comment:
Thes addition of this parametrization is the true change here. And it was added on all the tests. Github flags more stuff as big changes because the indentation changed on all tests because now the patching needs a parameter and we use `with patch(...):` rather than `@patch()` as it was before.